services/QuillLMS/app/services/pdf_file_builder.rb

```diff
@@ -0,0 +1,31 @@
+# frozen_string_literal: true
+
+class PdfFileBuilder < ApplicationService
```

Comment:
I'm not seeing this used anywhere in this PR, is there a follow-on PR to connect everything together or is their some implicit configuration with this gem that I'm not understanding?

Reply:
You are correct.  This is currently only working through the console.  A separate PR will help connect these pieces together.